docs/endpoint-release.md

```diff
@@ -25,3 +25,47 @@ The `/release` endpoint accepts several parameters:
 ### Caveats
 Note that the `/release` endpoint can be rather slow. This is because it involves running `git push` to a real repository, which in itself is a slow operation. Usually this takes about 1 second, but it highly depends on your Git Hosting Provider. This applies to all endpoints that have to write to the git repo (which is most of the endpoints).
 
+### CLI
+
+There is a Kubeprult command line client for communicating with the `/release` endpoint now at [`cli`](https://github.com/freiheit-com/kuberpult/tree/main/cli). The usage is as follows:
+
+```
+kuberpult-client release \
```

Comment:
it is called `kuberpult-cli` in another place, which one is correct?

Reply:
`kuberpult-client` is the correct name. I will update in the other place.